src/tests/unit/tests/bug-filing/github/__snapshots__/create-github-bug-filing-url.test.ts.snap

```diff
@@ -1,3 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`createGitHubBugFilingUrlTest creates url 1`] = `"test appendSuffixToUrl/new?title=test%20title&body=test%20issue%20details"`;
+exports[`createGitHubBugFilingUrlTest creates url 1`] = `"https://builded-url"`;
```

Comment:
Not blocking: seems strange to use snapshot for essentially testing against an expected string?

Reply:
It was a little more complex before... I can update to not use snapshot